### PR TITLE
[#355] [FEATURE] Email de bienvenue aux followers

### PR DESCRIPTION
**Pull request by florianEnoh**

### CODE REVIEW

**Comment by jbuget** [comment]
I've deployed this PR to http://367-mail-bienvenue-follower.pix.beta.gouv.fr. Please check it out

File: api/lib/application/followers/follower-controller.js [hunk 1]
```diff
@@ -2,8 +2,9 @@ const Boom = require('boom');
 const Follower = require('../../domain/models/data/follower');
 const EmailValidator = require('../../domain/services/email-validator');
 const followerSerializer = require('../../infrastructure/serializers/jsonapi/follower-serializer');
+const Mailjet = require('../../infrastructure/mailjet');
```
Comment (jbuget) [inline api/lib/application/followers/follower-controller.js]: `mailjet` (minuscule)

File: api/lib/infrastructure/mailjet.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+const mailjetConfig = require('../settings').mailjet;
+const Mailjet = require('node-mailjet').connect(mailjetConfig.apiKey, mailjetConfig.apiSecret);
```
Comment (jbuget) [inline api/lib/infrastructure/mailjet.js]: découper sur 2 lignes : 1) import 2) connect

File: api/lib/infrastructure/mailjet.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+const welcomeEmailTemplateId = '129291';
```
Comment (jbuget) [inline api/lib/infrastructure/mailjet.js]: rename `WELCOME_EMAIL_TEMPLATE_ID`

File: api/lib/infrastructure/mailjet.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  sendWelcomeEmail(receiverEmail){
+    return Mailjet
+      .post('send')
+      .request(_formatPayload(receiverEmail))
+      .then((result) => {
+        return result;
+      })
+      .catch((error) => {
+        return error;
+      });
```
Comment (jbuget) [inline api/lib/infrastructure/mailjet.js:27]: l.21 -> l.26 : lignes inutiles

File: api/lib/settings.js [hunk 5]
```diff
@@ -17,15 +17,26 @@ module.exports = (function () {
     airtable: {
       apiKey: 'keyEgu8JYhXaOhjbd',
       base: 'appHAIFk9u1qqglhX'
+    },
+
+    mailjet: {
```
Comment (jbuget) [inline api/lib/settings.js:22]: sortir ça "proprement"

File: api/lib/application/followers/follower-controller.js (hunk 1 above)
Comment (florianEnoh) [reply api/lib/application/followers/follower-controller.js]: Done :)

File: api/lib/infrastructure/mailjet.js (hunk 2 above)
Comment (florianEnoh) [reply api/lib/infrastructure/mailjet.js]: OK

File: api/lib/infrastructure/mailjet.js (hunk 3 above)
Comment (florianEnoh) [reply api/lib/infrastructure/mailjet.js]: (y)

File: api/lib/settings.js (hunk 5 above)
Comment (florianEnoh) [reply api/lib/settings.js:22]: Done

**Comment by jbuget** [comment]
I've deployed this PR to http://367-mail-bienvenue-follower.pix.beta.gouv.fr. Please check it out

File: api/lib/infrastructure/mailjet.js [hunk 6]
```diff
@@ -0,0 +1,29 @@
+const mailjetConfig = require('../settings').mailjet;
+const Mailjet = require('node-mailjet').connect(mailjetConfig.apiKey, mailjetConfig.apiSecret);
+const welcomeEmailTemplateId = '129291';
+
+function _formatPayload(email) {
+  return {
+    'FromEmail': 'communaute@pix.beta.gouv.fr',
+    'FromName': 'communauté pix',
```
Comment (jbuget) [inline api/lib/infrastructure/mailjet.js]: "Communauté Pix"

**Comment by jbuget** [comment]
I've deployed this PR to http://367-mail-bienvenue-follower.pix.beta.gouv.fr. Please check it out

**Comment by jbuget** [comment]
I've deployed this PR to http://367-mail-bienvenue-follower.pix.beta.gouv.fr. Please check it out

**Comment by jbuget** [comment]
I've deployed this PR to http://367-mail-bienvenue-follower.pix.beta.gouv.fr. Please check it out

**Comment by jbuget** [comment]
I've deployed this PR to http://367-mail-bienvenue-follower.pix.beta.gouv.fr. Please check it out

**Comment by jbuget** [comment]
I've deployed this PR to http://367-mail-bienvenue-follower.pix.beta.gouv.fr. Please check it out

**Comment by jbuget** [comment]
I've deployed this PR to http://367-mail-bienvenue-follower.pix.beta.gouv.fr. Please check it out

**Comment by jbuget** [comment]
I've deployed this PR to http://367-mail-bienvenue-follower.pix.beta.gouv.fr. Please check it out

**Pull request merged**